README.md

```diff
@@ -167,8 +165,12 @@ Internal use only targets include:
 
 ## Notes
 
-- If you're using a custom domain for GitHub Enterprise, be sure to include it through a `@-moz-document` rule (Firefox) or add it to the `Applies to` section in (Chrome). See
+- If you're using a custom domain for GitHub Enterprise, be sure to include it through a `@-moz-document` rule (Firefox) or add it to the `Applies to` section in (Chrome).
 
 Thanks to all our [contributors](./AUTHORS) so far! And thanks for the shoutout on the [JS Party](https://changelog.com/jsparty/20#transcript-71) podcast!
 
 [⬆️ UP](#toc)
+
+:before:after:
+:------:-----:
+:![image](https://user-images.githubusercontent.com/18245694/91337952-73cb2e80-e7d4-11ea-8e8c-5bb624c0af43.png):![image](https://user-images.githubusercontent.com/18245694/91338030-952c1a80-e7d4-11ea-948d-58b52c761b90.png):
```

Comment:
This seems unrelated.

Reply:
Yep! I tried to format a table in my text editor and forgot to remove it. 😅